presto-verifier/src/main/java/com/facebook/presto/verifier/framework/AbstractVerification.java

```diff
@@ -124,6 +124,13 @@ protected abstract R verify(
 
     protected void updateQueryInfo(QueryInfo.Builder queryInfo, Optional<QueryResult<V>> queryResult) {}
 
+    protected void updateQueryInfoWithQueryBundle(QueryInfo.Builder queryInfo, Optional<B> queryBundle)
```

Comment:
You can just call it `updateQueryInfo` since Java supports method overloading.

Reply:
I'm afraid that might not work because generic will be erased. Compiler complains:  'updateQueryInfo(Builder, Optional<QueryResult<V>>)' clashes with 'updateQueryInfo(Builder, Optional<B>)'; both methods have same erasure.